oprf-world-types/src/api.rs

```diff
@@ -0,0 +1,6 @@
+//! # API module
+//!
+//! Entry point for all API version modules.
+//!
+//! Currently exposes the version 1 OPRF API types under [`v1`].
```

Comment:
Probably this is not really needed to be scoped under v1? But fine to leave as is